.mdlrc

```diff
@@ -1 +1 @@
-rules "~MD013", "~MD032"
+rules "~MD013", "~MD032", "~MD041"
```

Comment:
We don't need this change since we never have a heading at the top of our pages.

Reply:
This is why it's needed, unless I'm missing something here?

Reply:
The 041 throws an error if we're missing first line H1 headings, but we don't have them or use them

Reply:
Yeah, this should stop it from throwing an error. ~MD041 is ignore this rule.

Reply:
041 is the rule itself. https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md041---first-line-in-a-file-should-be-a-top-level-heading

Reply:
There's a more complete `markdowlint.yaml` in the repo.